editor/scene_tree_dock.cpp

```diff
@@ -2921,7 +2921,7 @@ void SceneTreeDock::_create_remap_for_node(Node *p_node, Map<RES, RES> &r_remap)
 		if (v.is_ref()) {
 			RES res = v;
 			if (res.is_valid()) {
-				if (res->get_path() == "" && !r_remap.has(res)) {
+				if ((res->get_path() == "" || res->get_path().find("::") > -1) && !r_remap.has(res)) {
```

Comment:
On phone so I can't check, but I think we added some `is_builtin()` method somewhere (or similar name) to do exactly this kind of check.

Reply:
I found only `_is_built_in_script()`. Although maybe adding such method to Resource would make sense to avoid problems like this.

Reply:
Yeah that's the one I had in mind I guess. That's probably material for another PR but I agree that it could be added to `Resource` (or `Script`? more local, but would require casting before calling).